packages/styles/scss/components/masthead/_masthead-search.scss

```diff
@@ -93,14 +97,15 @@
   // search container
   .#{$prefix}--masthead__search,
   :host(#{$dds-prefix}-masthead-search) {
-    flex: 1;
     height: $carbon--spacing-09;
+    margin-left: 48px;
```

Comment:
just wondering if we should convert this to `rem` here. And alos, did you check if this works still if you increase the zoom level in the browser? I see some specific amounts being used here so just wanted to make sure.

Reply:
Yes, I just checked with zoom and it works! I'll convert to rem, good point!